libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -2198,16 +2326,20 @@ export const cryptocurrenciesById: Record<string, CryptoCurrency> = {
     id: "songbird",
     coinType: 60,
     name: "Songbird",
-    managerAppName: "Songbird",
+    managerAppName: "Ethereum",
     ticker: "SGB",
     scheme: "songbird",
-    color: "#000000",
-    family: "ethereum",
+    color: "#61ACD4",
+    family: "evm",
     ethereumLikeInfo: {
       baseChain: "mainnet",
       chainId: 19,
       networkId: 19,
-      hardfork: "berlin",
```

Comment:
Is removing this intended?

Reply:
Yes it was. The `hardfork` property is only used by the `ethereum` familly and necessary to a lib (`@ethereumjs/commmon`) that is not used by the `evm` familly, so it's just dead code at this point.